### PR TITLE
Versioning changes for PR $6151

### PR DESCRIPTION
**Pull request by praries880**
Versioning changes for PR $6151

### CODE REVIEW

**Comment by praries880** [comment]
sign job : https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/631/

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/859/

**Comment by praries880** [comment]
@markcowl  can you look at this PR

File: src/Storage/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,12 @@
 -->
 ## Current Release
 
+## Vearion 4.4.0
```
Comment (markcowl) [inline src/Storage/ChangeLog.md]: Version

Comment (praries880) [reply src/Storage/ChangeLog.md]: fixed

**Comment by markcowl** [comment]
Superseded by PR that also includes tooling changes

**Pull request closed without merging**